Misc/NEWS.d/next/Core and Builtins/2020-11-16-17-30-03.bpo-42316._DdmpQ.rst

```diff
@@ -0,0 +1 @@
+Allow an unparenthesized walrus in subscript indexes.
```

Comment:
Is this a regression? In that case the news entry should mention it

Reply:
Nope, this is no regression. It's new syntax.

Reply:
👍